.github/workflows/ci_cd.yml

```diff
@@ -10,7 +10,13 @@ on:
       image_tag:
         description: "The tag to assign to the images built in the workflow."
         type: string
-        required: true
+        required: false
+        default: ""
+      force_deploy:
+        description: "Whether to force GHCR publish and ECS deploy from a non-main branch."
+        type: boolean
```

Comment:
Doesn't have to be done in this PR, but some info on when we can do boolean vs string checks for `true`/`false` values would be really useful in our internal documentation for GitHub Actions!

Reply:
Yes, I will make a separate issue for that to keep this PR small and easy to review.

Reply:
Update: #3675